patroni/validator.py

```diff
@@ -1045,6 +1045,7 @@ def validate_watchdog_mode(value: Any) -> None:
     },
     Optional("tags"): {
         Optional("nofailover"): bool,
+        Optional("failover_priority"): int,
```

Comment:
what are minimum and maximum values?

Reply:
Minimum is 0, no maximum. I see that there is a more specific validator and have updated code accordingly